framework/mail/BaseMailer.php

```diff
@@ -94,7 +94,7 @@
     public function getComposer()
     {
         if (!is_object($this->_composer) || $this->_composer instanceof \Closure) {
-            if (is_array($this->_composer) && !isset($this->_composer['__class'])) {
+            if (!is_array($this->_composer) && !isset($this->_composer['__class'])) {
```

Comment:
Can you explain it a little more? This check does not make any sense right now - if `$this->_composer` is not an array, then `$this->_composer['__class']` will never exist. And it will never set default class for array without `__class` key, which seems to be the whole point of this piece of code.

Reply:
Hi @rob006 you are absolutely right, reviewing all the code and testing, it is expected that $this->_ composer is an array, and you only have to assign the class in case it does not exist, then we just initialize the variable _composer as an array.